src/natcap/invest/cli.py

```diff
@@ -13,85 +14,106 @@
 import warnings
 
 import natcap.invest
-from natcap.invest import datastack
-from natcap.invest import model_metadata
-from natcap.invest import spec_utils
-from natcap.invest import ui_server
-from natcap.invest import utils
 from pygeoprocessing.geoprocessing_core import GDALUseExceptions
+with GDALUseExceptions():
```

Comment:
I think we removed this recently.

Reply:
https://github.com/natcap/invest/blob/808a69796409ef72d32972e7e350fb719ffee89e/src/natcap/invest/cli.py#L15

Reply:
Ack, more merge issues!